orga/app/components/routes/authenticated/sco-students/list-items.js

```diff
@@ -25,6 +25,10 @@ export default class ListItems extends Component {
     return '.xml';
   }
 
+  get displayLearMoreAndLinkTemplate() {
```

Comment:
```suggestion
  get displayLearnMoreAndLinkTemplate() {
```
Il manque un n à "Learn" ;)